examples/post_training_quantization/openvino/anomaly_stfpm_quantize_with_accuracy_control/main.py

```diff
@@ -168,7 +168,7 @@ def transform_fn(data_item):
     # Benchmark performance, calculate compression rate and validate accuracy
 
     fp32_ir_path = f"{ROOT}/stfpm_fp32.xml"
-    ov.save_model(ov_model, fp32_ir_path, compress_to_fp16=False)
+    ov.save_model(ov_model, fp32_ir_path)
```

Comment:
OpenVINO saves model weights in FP16 precision by default. If you remove `compress_to_fp16=False` here, the model weights will be saved in FP16. This leads at a minimum to a change in the size of the model and a change in the output metrics of the example. I would suggest not to remove `compress_to_fp16=False` for FP32 models here in another examples, so as not to change the behavior of the example.

Reply:
Sorry, I've been busy all week, finally done it. Can you check, please?